crates/bevy_mesh/src/primitives/extrusion.rs

```diff
@@ -72,7 +72,7 @@ impl PerimeterSegment {
 /// ## Warning
 ///
 /// By implementing this trait you guarantee that the `primitive_topology` of the mesh returned by
-/// this builder is [`PrimitiveTopology::TriangleList`](wgpu::PrimitiveTopology::TriangleList)
+/// this builder is [`PrimitiveTopology::TriangleList`](PrimitiveTopology::TriangleList)
```

Comment:
```suggestion
/// this builder is [`PrimitiveTopology::TriangleList`]
```

This should fix CI! `PrimitiveTopology` is already imported, so the "link" is redundant.

Reply:
+ The failure on the dependencies check is unrelated, so there's not much to be done about that.

Reply:
thanks!